packages/asset-buyer/src/order_providers/standard_relayer_api_order_provider.ts

```diff
@@ -111,4 +111,32 @@ export class StandardRelayerAPIOrderProvider implements OrderProvider {
             }
         });
     }
+    /**
+     * Given a maker asset data string, return all availabled paired taker asset data strings.
+     * @param   makerAssetData   A string representing the maker asset data.
+     * @return  An array of asset data strings that can be used to purchased makerAssetData.
+     */
+    public async getAvailableTakerAssetDatasAsync(makerAssetData: string): Promise<string[]> {
+        // Return a maximum of 1000 asset datas
+        const maxPerPage = 1000;
```

Comment:
This is hidden from the caller and not immediately obvious. Let's make it a optional param that details to 1000.